st2client/tests/unit/test_auth.py

```diff
@@ -154,6 +157,14 @@ def runTest(self, mock_gp):
 
         self.shell.run(args)
 
+        # TODO(mierdin): This tests that this particular command sends X-Auth-Token but you should
+        # also test other commands after this token has been installed
+        kwargs = {
+            'headers': {'X-Auth-Token': self.TOKEN['token'], 'content-type': 'application/json'},
```

Comment:
Alright, I belive this should do it - cd89193fd69bda32a1f9591faa1e9ea6f8e953a0, 545c7baef5225e9e9c0a10d97c25dec3b822c87c.